redhat/ubi8-openjdk-11.yaml

```diff
@@ -1,5 +1,3 @@
-from: "registry.redhat.io/ubi8/ubi-minimal"
```

Comment:
Are you sure this is right? AFAIK, only registry.redhat.io URLs are working in OSBS, no?

Reply:
registry.access.redhat.com has now been added to the whitelist in OSBS. I performed a test scratch build to confirm

Reply:
OK.